python/nav/web/api/v1/auth.py

```diff
@@ -60,12 +60,18 @@ def authenticate(self, request):
             return request.account, None
 
 
-class LoggedInPermission(BasePermission):
-    """Checks if the user is logged in"""
+class ReadOnlyNonAdminPermission(BasePermission):
+    """
+    Gives non-admin users read-only permission and admin users read-write permission
+    """
 
     def has_permission(self, request, _view):
-        """If user is logged in, it is authorized"""
-        return not request.account.is_default_account()
+        """If user is logged in and it is a safe method, it is authorized"""
+        if request.method in SAFE_METHODS and not request.account.is_default_account():
+            return True
+        elif request.account.is_admin():
```

Comment:
You don't need the `elif`, just the `if`, since you return early.

Reply:
I knew there was something off about it! Good that code reviews exist!